domain/schema/controller.go

```diff
@@ -598,3 +599,13 @@ CREATE TABLE model_config_defaults (
 )
 `)
 }
+
+func flagSchema() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE flag (
+    name TEXT PRIMARY KEY,
+    -- SQLite doesn't have a boolean type, so we use an integer instead.
```

Comment:
Call it `BOOLEAN` anyway, even though it's an integer under the hood. We've used it a bunch already.